Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -2424,14 +2424,44 @@ Optional<StyleProperties> StyleComputer::compute_style_impl(DOM::Element& elemen
         start_needed_transitions(*previous_style, style, element, pseudo_element);
     }
 
-    // Remove color names from CSS color values. This is needed because computed values cannot be named colors.
+    // "The computed value of a <position> is a pair of offsets (horizontal and vertical),
+    //  each given as a computed <length-percentage> value,
+    //  representing the distance between the left edges and top edges (respectively)
+    //  of the alignment subject and alignment container."
```

Comment:
Can we do this in `StyleProperties` instead? That's normally where we massage specified values into computed values.

Reply:
Can `StyleProperties.cpp` be used to define how style values of a specific type (i.e. all `<position>` values) are computed or is it limited to individual properties (i.e. `background-position` and `object-position`)?
More specifically, does the position need to be calculated separately in both `StyleProperties::object_position()` and a new `StyleProperties::background_position()` (and all future uses of positions)?